lib/remap-functions/src/parse_grok.rs

```diff
@@ -103,15 +103,15 @@ impl Expression for ParseGrokFn {
 
                 Ok(Value::from(result))
```

Comment:
This still seems to return a `Map`, not an `Array`.

Reply:
> This still seems to return a `Map`, not an `Array`.

Correct. It was the type_def that was wrong.